collections/_article/dev-snapshot-godot-4-5-dev-1.md

```diff
@@ -0,0 +1,137 @@
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/dev-snapshot-godot-4-5-dev-1/preload-uid.webm?1" type="video/webm">
+</video>
+
+### Allow selecting multiple remote nodes at runtime
+
+A long-awaited QOL addition to the editor experience has finally arrived! Thanks to [Michael Alexsander](https://github.com/YeldhamDev), developers now have the ability to select multiple nodes in a runtime context! Check out their pull request [GH-99680](https://github.com/godotengine/godot/pull/99680) for more information on how this was integrated.
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/dev-snapshot-godot-4-5-dev-1/multiple-nodes.webm?1" type="video/webm">
+</video>
+
+### And more!
+
+There are too many exciting changes to list them all here, but here's a curated selection:
+
+- 2D: Improve usability of `Camera2D` ([GH-101427](https://github.com/godotengine/godot/pull/101427)).
+- 3D: Fix `Camera3D` gizmo representation to accurately reflect FOV ([GH-101884](https://github.com/godotengine/godot/pull/101884)).
+- 3D: Use physical keys for the Q/W/E/R editor shortcuts ([GH-103533](https://github.com/godotengine/godot/pull/103533)).
+- Android: Add `linux-bionic` RID export option ([GH-97908](https://github.com/godotengine/godot/pull/97908)).
+- Android: Add a `TouchActionsPanel` to editor ([GH-100339](https://github.com/godotengine/godot/pull/100339)).
+- Android: Enable support for volume button events ([GH-102984](https://github.com/godotengine/godot/pull/102984)).
+- Animation: Support hiding functions calls in Method Tracks ([GH-96421](https://github.com/godotengine/godot/pull/96421)).
+- Core: Add `scene_changed` signal to `SceneTree` ([GH-102986](https://github.com/godotengine/godot/pull/102986)).
+- Core: Add DDS image load and save functionality ([GH-101994](https://github.com/godotengine/godot/pull/101994)).
+- Core: Don't duplicate internal nodes ([GH-89442](https://github.com/godotengine/godot/pull/89442)).
+- Core: Implement `get_size` and `get_access_time` methods to `FileAccess` ([GH-83538](https://github.com/godotengine/godot/pull/83538)).
+- Debugger: Allow locating VRAM resource by double-clicking ([GH-103949](https://github.com/godotengine/godot/pull/103949)).
+- Documentation: Overhaul `Node3D` documentation ([GH-87440](https://github.com/godotengine/godot/pull/87440)).
+- Editor: Add option to copy a file's name in the FileSystem dock ([GH-96536](https://github.com/godotengine/godot/pull/96536)).
+- Editor: Allow ignoring debugger error breaks ([GH-77015](https://github.com/godotengine/godot/pull/77015)).
+- Editor: Convert `compress_native_libraries` to a basic export option ([GH-104301](https://github.com/godotengine/godot/pull/104301)).
+- Editor: Don't save unnecessarily with save_before_running ([GH-90034](https://github.com/godotengine/godot/pull/90034)).
+- Editor: Don't save unnecessarily with save_before_running ([GH-90034](https://github.com/godotengine/godot/pull/90034)).
```

Comment:
Duplicate, and formatting. It seems the PR title already has code markup, so it should be in the interactive changelog.

You can copy from the Markdown/Unified view so you get all the correct titles and formatting:
![image](https://github.com/user-attachments/assets/19480644-2e1d-4ba5-8df7-7e3b22eea794)

Though I guess with the new workflow flagging what to mention in the merge queue it's a bit trickier.

```suggestion
- Editor: Don't save unnecessarily with `save_before_running` ([GH-90034](https://github.com/godotengine/godot/pull/90034)).
```